src/dags/common/dataschema.py

```diff
@@ -0,0 +1,50 @@
+import json, jsonschema
+from .utils import schema_fetch_url_file
+from schematools.provenance.create import ProvenaceIteration
+
+
+def get_column_names_for_ogr2ogr_sql_select(
+    dataschema, file_to_save, source_filename, source_missing_cols=(), skip_cols=()
```

Comment:
Nog een typische python instinker. De kwargs die nu als default een tuple hebben kunnen problemen opleveren. Deze tuples worden eenmalig aangemaakt bij de python byte compilatie. Roep je de functie meermalen aan, dan kunnen de kwargs de waarden van de vorige aanroep bevatten. De standaard oplossing is om hier als waarde een `None` te gebruiken. In de code van de functie check je dan met een `if source_missing_cols is not None`.

Reply:
Super Jan! Altijd welkom zulke pointers. Daar leer ik van :-) Ik heb het ook nagespeeld en snap wat je bedoeld.